pdm-api/src/main/java/com/teamthree/pdmapi/persistence/CollectionDatabaseDAO.java

```diff
@@ -174,5 +180,60 @@ public Book[] searchBook(String collectionId, String str) {
         }
         return (Book[]) books.toArray();
     }
-    
+
+    /**
+     * Updates a collection's name in the database
+     */
+    @Override
+    public boolean updateCollectionName(String collectionId, String newName) {
+        String query = "UPDATE Collection SET collectionName = '" + newName + "'  WHERE collectionId = '" + collectionId;
+        try{
+            Statement stmt = connHandler.getConnection(false).createStatement();
+            stmt.executeUpdate(query);
+        } catch(SQLException e) {
+            e.printStackTrace();
+            return false;
+        } finally {
+            connHandler.closeConnection();
+        }
+        return true;
+    }
+
+    /**
+     * Displays details of collections in the database
+     * Includes collection’s name,number of books and total length of the books in the collection
+     * Displat collections by name in ascending order
+     */
+    @Override
+    public List<Map<String, Object>> getCollectionsWithDetails(String accountId) {
```

Comment:
We probably don't need this since we already have other functions that can get these values. It's also generally not good to have a function that takes care of so much by itself. We can leave it in, but it probably won't get used